docker-for-windows/index.md

```diff
@@ -443,64 +441,68 @@ to change context so that `kubectl` is pointing to `docker-for-desktop`:
 > kubectl config use-context docker-for-desktop
 ```
 
-You can also change it through the Docker Desktop for Windows menu:
+You can also change it through the Docker Desktop menu:
 
-![Change Kubernetes Context](images/docker-menu-context-switch.png){:width="600px"}
+![Change Kubernetes Context](images/docker-menu-context-switch.png){:width="450px"}
 
 If you installed `kubectl` by another method, and
 experience conflicts, remove it.
 
 - To enable Kubernetes support and install a standalone instance of Kubernetes
   running as a Docker container, select **Enable Kubernetes** and click the
-  **Apply and restart** button.
+  **Apply** button.
 
-  An Internet connection is required. Images required to run the Kubernetes
-  server are downloaded and instantiated as containers, and the
-  > Program Files\Docker\Docker\Resources\bin\kubectl.exe` command is installed.
+  Note that you must have Internet connection. Images required to run the Kubernetes server are downloaded and instantiated as containers, and the kubectl.exe command is installed in the path.
```

Comment:
Should be "must have an Internet connection". But actually, I'm not sure if we removed this requirement. We certainly removed it for Enterprise. Needs checking.

Reply:
Posted a question in the desktop-dev channel. I'll update when I receive a confirmation.

Reply:
Removed the requirement as per Mathieu's confirmation.